less/advancement.less

```
@@ -205,6 +205,23 @@
     }
   }
 
+  /* ----------------------------------------- */
+  /*  Size                                     */
+  /* ----------------------------------------- */
+  &.size {
+    .trait-list {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 0.25rem;
+      padding: 0;
+      list-style: none;
+
+      li {
+        flex: 1 1 40%;
+      }
```

CSS grid might make more sense here rather than using a flex-basis of 40% with flex-wrap.

```suggestion
      display: grid;
      grid-template-columns: 1fr 1fr;
      gap: 0.25rem;
      padding: 0;
      list-style: none;
```